pages/api/business/listing/pushQueuedListing.js

```diff
@@ -59,15 +71,26 @@ const handler = async (req, res) => {
         },
       });
 
+      // Log relevant information
+      console.log("New Listing:", newListing);
+
       await prisma.queuedListing.delete({
         where: { id: data.id },
       });
 
-      return newListing;
-    });
+      // Commit the transaction
+      await prisma.$executeRaw`COMMIT`;
```

Comment:
is it fine to mix up raw with regular commands?

Reply:
It should be, provided regular operations are scoped entirely inside raw commands or vice versa